env/production/config.json

```diff
@@ -5,5 +5,7 @@
   "OAUTH2_LOGOUT_URL": "https://login.nextstrain.org/logout",
   "COGNITO_USER_POOL_ID": "us-east-1_Cg5rcTged",
   "OIDC_USERNAME_CLAIM": "cognito:username",
-  "OIDC_GROUPS_CLAIM": "cognito:groups"
+  "OIDC_GROUPS_CLAIM": "cognito:groups",
+  "SESSION_COOKIE_DOMAIN": "nextstrain.org",
+  "GROUPS_DATA_FILE": "groups.json"
```

Comment:
Suggestion: add these to the [infrastructure](https://github.com/nextstrain/nextstrain.org/blob/dc8500adc81aba198c474b144aaae0c9a7b3baa8/docs/infrastructure.md?plain=1#L71-L91) doc page.

Reply:
Done.